meson.build

```diff
@@ -36,10 +38,16 @@ foreach s : sources_list
     sources += 'source/' + s + '.d'
 endforeach
 
-add_project_arguments([
-    '-preview=dip1008',
-    '-lowmem',
-], language: 'd')
+if dc.get_id() == 'gcc'
+    add_project_arguments([
+        '-fpreview=dip1008',
```

Comment:
did you want to add (GDC's equivalent of) `-lowmem` here too?

Reply:
My GDC12 does not seem to have anything resembling that - do you know the right flag? (if it were there, it would likely be `-flowmem`, but that doesn't exist, so I guess this frontend feature is not wired up in GDC).

Reply:
Its not a language feature flag, so I'm not really surprised you can't find it under`-f`. Perhaps @ibuclaw can help here.

Reply:
I don't see the point of having the switch, either it's always on, or it's always off, such as a GC scan bug that leads to unexpected freeing on target X.

Reply:
Since meson does a split build by default (unity builds aren't implemented for D yet), running out of memory is fairly rare (even though it's possible, I've seed builds of some of my own projects use more than 8GB of memory with split builds due to `ninja` running many LDC instances in parallel).
The problem doesn't happen on Debian's builders and I've never seen it locally as well though :-)

Reply:
OK